index.html

```diff
@@ -1,417 +1,245 @@
-                    <button type="button">See Project</button>
-                </div>
-            </div>
-
-            <div class="project">
-                <div class="projectHeader">
-                    <h2>Profesional Art Printing Data</h2>
-                </div>
-
-                <div class="projectBody">
-                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been
-                        the industry's standard</p>
-                    <ul>
-                        <li class="projectHTML">
-                            <ul>
-                                <li>html</li>
-                            </ul>
-                        </li>
-                        <li class="projectBootstrap">
-                            <ul>
-                                <li>bootstrap</li>
-                            </ul>
-                        </li>
-                        <li class="projectRuby">
-                            <ul>
-                                <li>ruby</li>
-                            </ul>
-                        </li>
-                    </ul>
-                </div>
-                <div class="projectButton">
-                    <button type="button">See Project</button>
-                </div>
-            </div>
-
-            <div class="project">
-                <div class="projectHeader">
-                    <h2>Profesional Art Printing Data</h2>
-                </div>
-
-                <div class="projectBody">
-                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been
-                        the industry's standard</p>
-                    <ul>
-                        <li class="projectHTML">
-                            <ul>
-                                <li>html</li>
-                            </ul>
-                        </li>
-                        <li class="projectBootstrap">
-                            <ul>
-                                <li>bootstrap</li>
-                            </ul>
-                        </li>
-                        <li class="projectRuby">
-                            <ul>
-                                <li>ruby</li>
-                            </ul>
-                        </li>
-                    </ul>
-                </div>
-                <div class="projectButton">
-                    <button type="button">See Project</button>
-                </div>
+<header>    
+    <nav class="nav-bar" >
+        <div class="link link-icon logo"><a>Benard Ngetich </a> </div>
+        <ul class="menu-list ul-list horizontal-list">
+            <li class="list-item-text"><a href="#" class="link link-text">Portfolio</a></li>
+            <li class="list-item-text"><a href="#" class="link link-text">About</a></li>
+            <li class="list-item-text"><a href="#" class="link link-text">Contact</a></li>
```

Comment:
The nav is the element of your HTML page that redirects you to other pages/sections of your website. Taking this into account, please edit your `a` tags, making them direct to the corresponding sections. This will make user navigation easier. 😉
> Tip: It isn't a good practice to leave empty `a` tags on your project. Try to give all of them a purpose. 😃